### DISCUSSION TITLE
Shared components across Zones

### DISCUSSION THREAD

**Discussion by kennethacohen** [vercel/next.js#19851] [Help]
I'm looking to split up my current app into separate Zones, but it would be nice for each of the separate zones to share a components folder. Is there a recommended pattern for this, or should I not move forward with this approach?

**Comment by dpidan** [comment]
Did you also split up your apps into separate packages - turning your site into a monorepo?
You'd probably be best off doing that and then creating a `components` package that you import into each app.
